x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -7,6 +7,18 @@
 
 import { i18n } from '@kbn/i18n';
 
+export const ELSER_MODEL_ID = '.elser_model_2';
+export const ELSER_LINUX_OPTIMIZED_MODEL_ID = '.elser_model_2_linux-x86_64';
+export const E5_MODEL_ID = '.multilingual-e5-small';
+export const E5_LINUX_OPTIMIZED_MODEL_ID = '.multilingual-e5-small_linux-x86_64';
+export const LANG_IDENT_MODEL_ID = 'lang_ident_model_1';
+export const ELSER_ID_V1 = '.elser_model_1' as const;
+
+export const getLatestElserModel = () => ELSER_LINUX_OPTIMIZED_MODEL_ID;
+export const getLatestE5Model = () => E5_LINUX_OPTIMIZED_MODEL_ID;
```

Comment:
As we provide models based on the cluster architecture, I reckon you should use [getCuratedModel](https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/models/model_management/models_provider.ts#L509) instead of this static definition.

Reply:
Done in https://github.com/elastic/kibana/pull/187154/commits/9df4b804b06244bb2f9f4cacb338a8a0abe7c931